lib/grape-rabl/render.rb

```diff
@@ -1,8 +1,8 @@
 module GrapeRabl
   module Render
     def render(options = {})
-      env['api.endpoint'].options[:route_options][:rabl] = options.delete(:rabl) if options.include?(:rabl)
-      env['api.endpoint'].options[:route_options][:rabl_locals] = options.delete(:locals)
+      env['api.tilt.rabl'] = options.delete(:rabl) if options.include?(:rabl)
```

Comment:
Key seems to be the recommended de-factor standard of doing this Ruby-style, it is shorter than `has_key?` or `include?` by 4 characters and it clearly states that this is a Hash, not any other structure.